src/mol-gl/shader/postprocessing.frag.ts

```diff
@@ -152,7 +153,10 @@ void main(void) {
             }
         }
     #endif
-
+    if (uGamma > 1.0)
```

Comment:
Adding something like gamma correction needs more thought. Would using the exposure param in the renderer work for you for now?

Reply:
yes, I removed the gamma changes.